services/web/server/src/simcore_service_webserver/catalog.py

```diff
@@ -76,8 +76,12 @@ async def _reverse_proxy_handler(request: web.Request):
     # forward request
     session = get_client_session(request.app)
 
+    # add product to headers @crespov, here where the product shall come in
```

Comment:
this comment is not that clear for me.

Reply:
@GitHK I will take care of this in my PR. I tis a not he added so that i recall it. We spoke about it. No worries :-)

Reply:
ok so with the latest modifications, the file-picker and group-node shall also disappear.
and the comment is for pedro and his PR